### PR TITLE
[studio-iu] Create Site Dialog Polish #3535

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/3532

### CODE REVIEW

File: ui/app/src/components/MobileStepper.tsx [hunk 1]
```diff
@@ -92,6 +92,10 @@ const MobileStepper = React.forwardRef(function MobileStepper(props:MobileSteppe
     ...other
   } = props;
 
+  const capitalize = (s: string) => {
```
Comment (rart) [inline ui/app/src/components/MobileStepper.tsx]: Function already exists in utils/string

File: ui/app/src/components/BlueprintCard.tsx [hunk 2]
```diff
@@ -136,34 +154,38 @@ const messages = defineMessages({
   crafterCms: {
     id: 'common.crafterCMS',
     defaultMessage: 'Crafter CMS'
-  }
+  },
```
Comment (rart) [inline ui/app/src/components/BlueprintCard.tsx:161]: Keep these translation keys scoped to this dialog.

**Review by rart — APPROVED** [review]

File: ui/app/src/components/ConfirmDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import Dialog from "@material-ui/core/Dialog";
```
Comment (sumerjabri) [inline ui/app/src/components/ConfirmDialog.tsx:1]: Always add copyright.

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Add copyright to all files.

**Comment by sumerjabri** [comment]
@yureii00 I don't see the copyright added.

File: ui/app/src/components/ConfirmDialog.tsx (hunk 3 above)
Comment (sumerjabri) [reply ui/app/src/components/ConfirmDialog.tsx:1]: Copyright is still missing.

**Pull request merged**